### PR TITLE
Added functions get and count inside dbstorage and filestorage

### PR DESCRIPTION
**Pull request by DiegoAlar**
Hi David, please review files filestorage and dbstorage. I added two functionc (get and count) review the code and provide me with some feedback. Thanks!

### CODE REVIEW

File: models/engine/file_storage.py [hunk 1]
```diff
@@ -68,3 +68,16 @@ def delete(self, obj=None):
...
+        """ Retrieves an object given its class name and id """
+        if cls and id:
+            objs = self.all(cls)
+            key = cls.__name__ + '.' + id
+            return objs.get(key)
+        else:
+            return None
+
+    def count(self, cls=None):
+
```
Comment (dagute) [inline models/engine/file_storage.py]: Nice code Diego but you forget to  comment your function.

**Review by dagute — CHANGES_REQUESTED** [review]
Please add comments on your function.

File: models/engine/file_storage.py (hunk 1 above)
Comment (DiegoAlar) [reply models/engine/file_storage.py]: ALright, I'll check now!

File: models/engine/file_storage.py [hunk 2]
```diff
@@ -79,5 +79,5 @@ def get(self, cls, id):
             return None
 
     def count(self, cls=None):
-        
+        """ Counts the number of bjects """
```
Comment (dagute) [inline models/engine/file_storage.py:82]: done.

**Review by dagute — APPROVED** [review]
done.

**Pull request merged**